controllers/vspherecluster_controller.go

```diff
@@ -313,6 +320,59 @@ func (r *VSphereClusterReconciler) reconcileCloudProvider(ctx *context.ClusterCo
 	return nil
 }
 
+func (r *VSphereClusterReconciler) reconcileCSIDriver(ctx *context.ClusterContext) error {
```

Comment:
Hi @andrewsykim,

As much as I debated you about the way you're doing this, it's damned beautiful code sir!